content/blog/1-pack-up-the-gears-for-your-VR-trip.md

```diff
@@ -0,0 +1,43 @@
++++
+id = "0007"
+date = "2016-09-16T15:30:00+00:00"
+title = "Crossing realities: A VR journey"
+description = "TODO"
+author = "Quentin"
+tags = [ "Unity" "Cardboard" "VR" "tutorial" ]
++++
+
+## Pack up the gears for your VR trip
+
+The two main things you need to start developing VR apps is a Cardboard headset and Unity. You can get the first one for less than 10$ by ordering online and the second one for free.
+### Choosing your weapon:
+When choosing your Cardboard make sure that your model either have a magnetic trigger or a hole that let you tap on the screen. Some Cardboard models don't have any of those which limits a lot the possible interactions in game. Also check that it has a QR code. This QR code contains information to correct the distortion induced by the lenses.
+
+### Unity setup
+
+#### Install Unity
+To get Unity just go on the official website (https://unity3d.com/) and download the “Personal” edition. When installing you will be asked which components you'd like to install, I use an Android phone for my builds so I tick the “Android Build Support” checkbox. If you want to build your apps for IOS tick “iOS Build Support”.
+
+![article-img-centered](https://raw.githubusercontent.com/Tengio/tengio.com/master/static/img/blog/0007/1_0_unity_install_components_selection.png "Unity install")
+
+While Unity is installing you can create your compulsory Unity account (https://id.unity.com/account/new).
```

Comment:
to start developing VR apps is (ARE)

that your model either have (HAS) a

or a hole that let(S)

have any of those which limits (LIMIT)

interactions in game(S)